### PR TITLE
Speed up BPs at genesis startup

### PR DESCRIPTION
**Pull request by leventeliu**

### CODE REVIEW

**Review by xq262144 — APPROVED** [review]
Approve

File: blockproducer/rpc.go [hunk 1]
```diff
@@ -188,6 +188,31 @@ func WaitDatabaseCreation(
...
+		}
+		resp = &types.FetchTxBillingResp{}
+	)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-ticker.C:
+			if err = rpc.RequestBP(
+				route.MCCFetchBlock.String(), req, resp,
+			); err == nil || strings.Contains(err.Error(), "can't find service") {
```
Comment (laodouya) [inline blockproducer/rpc.go]: Make it CONST

Comment (laodouya) [reply blockproducer/rpc.go]: or an error type

**Review by laodouya — DISMISSED** [review]

File: blockproducer/rpc.go (hunk 1 above)
Comment (leventeliu) [reply blockproducer/rpc.go]: > 
> 
> or an error type

Nope, it's just an error literal like that in the RPC library...

**Review by zeqing-guo — APPROVED** [review]
Approve

**Pull request merged**